providers/axfrddns/axfrddnsProvider.go

```diff
@@ -133,14 +135,22 @@ func initAxfrDdns(config map[string]string, providermeta json.RawMessage) (provi
 	if err != nil {
 		return nil, err
 	}
+	switch strings.ToLower(strings.TrimSpace(config["buggy-cname"])) {
```

Comment:
`documentation/providers/axfrddns.md` should explain the `buggy-cname` setting.

Reply:
Done